GANDLF/training_loop.py

```diff
@@ -299,6 +303,11 @@ def validate_network(model, valid_dataloader, params):
         average_epoch_valid_metric[metric] = total_epoch_valid_metric[metric] / len(
             valid_dataloader
         )
+    
+    if params['scheduler'] == "reduce-on-plateau":
```

Comment:
exactly what i wanted to avoid

Reply:
I understand - but how do we get around this, then? Completely remove all schedulers?